airflow/providers/cncf/kubernetes/hooks/kubernetes.py

```diff
@@ -502,6 +503,30 @@ def create_job(
             raise e
         return resp
 
+    def get_job(self, job_name: str, namespace: str) -> V1Job:
+        """Get Job of specified name from Google Cloud.
+
+        :param job_name: Name of Job to fetch.
+        :param namespace: Namespace of the Job.
+        :return: Job object
+        """
+        return self.batch_v1_client.read_namespaced_job(name=job_name, namespace=namespace, pretty=True)
```

Comment:
`pretty` will print something, no?

Reply:
According to documentation, this parameter is indeed for outputting everything pretty :)
https://github.com/kubernetes-client/python/blob/master/kubernetes/client/api/batch_v1_api.py#L2674
i have set this one to True, since i am using those api calls for output of the information